pages/authorDashboard/PKPAuthorDashboardHandler.inc.php

```diff
@@ -216,19 +216,6 @@ public function setupTemplate($request)
         $submissionApiUrl = $request->getDispatcher()->url($request, PKPApplication::ROUTE_API, $submissionContext->getData('urlPath'), 'submissions/' . $submission->getId());
         $latestPublicationApiUrl = $request->getDispatcher()->url($request, PKPApplication::ROUTE_API, $submissionContext->getData('urlPath'), 'submissions/' . $submission->getId() . '/publications/' . $latestPublication->getId());
 
-        $contributorsGridUrl = $request->getDispatcher()->url(
-            $request,
-            PKPApplication::ROUTE_COMPONENT,
-            null,
-            'grid.users.author.AuthorGridHandler',
-            'fetchGrid',
-            null,
-            [
-                'submissionId' => $submission->getId(),
-                'publicationId' => '__publicationId__',
-            ]
-        );
```

Comment:
I totally forgot to test the author dashboard. I'll do that for the next round of review.